templates/controllers/grid/queries/form/queryForm.tpl

```diff
@@ -26,10 +26,20 @@
 
 	{include file="controllers/notification/inPlaceNotification.tpl" notificationId="queryFormNotification"}
 
-	{fbvFormArea id="queryUsersArea"}
-		{url|assign:queryUsersUrl router=$smarty.const.ROUTE_COMPONENT component="listbuilder.users.QueryUsersListbuilderHandler" op="fetch" queryId=$queryId params=$actionArgs escape=false}
-		{load_url_in_div id="queryUsersContainer" url=$queryUsersUrl}
-	{/fbvFormArea}
+	{fbvFormSection class="query_participants" title="editor.submission.stageParticipants" required="true"}
+		<ul>
+			{foreach from=$participantOptions item=participantOption}
+				<li>
+					{assign var="inputId" value="queryForm-user-"|concat:$participantOption.user->getId()}
+					<label for="{$inputId}">
+						<input type="checkbox" name="users[]" id="{$inputId}" value="{$participantOption.user->getId()}"{if $participantOption.isParticipant} checked="checked"{/if}>
```

Comment:
Actually I didn't even think about that!

I just tried to switch over to the checkbox group fbvElement, but it looks like that strips out HTML tags from the label. That means I can't do the `<span>` wrapping to style the user names and role names separately.

So I'll leave this in, but generally you're right using the `fbvElement` options is better. :)